### DISCUSSION TITLE
Support for Twitter Polls

### DISCUSSION THREAD

**Discussion by PavanaU** [tweepy/tweepy#1510] [Questions] [answered]
Hi Tweepy devs,

Will the Support to create Twitter poll be added?

**Comment by Harmon758** [comment, marked as answer]
This isn't supported by Twitter's API (outside of the Ads API) yet.
See https://twittercommunity.com/t/how-to-create-twitter-poll-card/38368, https://twittercommunity.com/t/polls/75456, and https://twittercommunity.com/t/designing-the-new-engage-endpoints/142766,
